banzai/settings.py

```diff
@@ -78,5 +78,10 @@
 
 TELESCOPE_FILENAME_FUNCTION = 'banzai.utils.file_utils.telescope_to_filename'
 
+ARCHIVE_API_ROOT = os.getenv('API_ROOT', 'https://archive-api.lco.global/frames')
```

Comment:
Are these environment variables not going to be set? It might be safer to default to non-production values.

Reply:
Removed the default. We are always going to set these.